### PR TITLE
Creates Stocks Screener SDK Guide

### PR DESCRIPTION
**Pull request by deeleeramone**
new file created.

### CODE REVIEW

**Review by Chavithra — APPROVED** [review]
Looks ok to me.

File: website/content/sdk/guides/intros/stocks/stocks-screener.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+
+Price/Free Cash Flow = Any
+```
+
+## Examples
+
+To make a screener with only a few filters, it should look something like this:
+
+```console
+# Author of preset: Danglewood
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/stocks-screener.md:57]: Let's change this OpenBB

File: website/content/sdk/guides/intros/stocks/stocks-screener.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+|  0 | GEO      |      0.1389 |       0.4005 |       0.4589 |      0.6723 |      0.3955 |     0.5342 |         0.0574 |         0.0558 |     2.5 |      2.09e+06 |         4.23 |   11.89 |   0.1623 |      8.86013e+06 |
+|  1 | EBS      |     -0.0059 |      -0.4513 |      -0.5136 |     -0.6429 |     -0.7204 |    -0.7292 |         0.0452 |         0.0827 |     2.7 | 753610        |         2.91 |   11.77 |   0.0138 |      2.19107e+06 |
+|  2 | TME      |      0.1871 |       0.7984 |       0.3721 |      0.5789 |     -0.0531 |    -0.0365 |         0.0681 |         0.0756 |     2.3 |      9.95e+06 |         2.2  |    6.6  |   0.0611 |      2.18486e+07 |
+|  3 | QFIN     |      0.0874 |       0.4331 |      -0.0738 |     -0.0744 |     -0.389  |    -0.365  |         0.0472 |         0.0787 |     1.4 | 997070        |         1.62 |   14.56 |   0.0849 |      1.61386e+06 |
+|  4 | ATHM     |      0.0229 |       0.0391 |      -0.1772 |     -0.2256 |     -0.1971 |    -0.0455 |         0.0492 |         0.0579 |     2.2 | 577050        |         1.6  |   28.14 |   0.0484 | 922812           |
+...continued...
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/stocks-screener.md:94]: Please remove this continued statements and either do `...` or put continued at the bottom. It messes up the table.

File: website/content/sdk/guides/intros/stocks/stocks-screener.md [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+|  8 | PINC     | Premier, Inc.                         | Healthcare             | Health Information Services    | USA       |    3.92e+09  |  21.52 |   33.2  |   0.0097 | 373171           |
+|  9 | AEP      | American Electric Power Company, Inc. | Utilities              | Utilities - Regulated Electric | USA       |    4.791e+10 |  19.63 |   94.65 |  -0.006  |      2.04444e+06 |
+| 10 | LRN      | Stride, Inc.                          | Consumer Defensive     | Education & Training Services  | USA       |    1.54e+09  |  16.74 |   35.8  |  -0.0047 | 270842           |
+| 11 | AMN      | AMN Healthcare Services, Inc.         | Healthcare             | Medical Care Facilities        | USA       |    5.4e+09   |  11.76 |  122.01 |  -0.0078 | 264138           |
+
+### Ownership
+
+When `data_type = 'ownership'`, data presented are statistics for the general float, insider, institutional, and the short ratio.
+
+openbb.stocks.screener.screener_data(preset_loaded='sdk_guide_preset.ini', data_type = 'ownership', limit = 100)
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/stocks-screener.md:158]: This piece of code is missing a code block.

File: website/content/sdk/guides/intros/stocks/stocks-screener.md [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+```
+
+|    | Ticker   | Company               | Sector                 | Industry                       | Country   |   Market Cap |   P/E |   Price |   Change |   Volume |
+|---:|:---------|:----------------------|:-----------------------|:-------------------------------|:----------|-------------:|------:|--------:|---------:|---------:|
+|  0 | AAPL     | Apple Inc.            | Technology             | Consumer Electronics           | USA       |  2.23252e+12 | 23.14 |  141.17 |  -0.0211 | 83578852 |
+|  1 | MSFT     | Microsoft Corporation | Technology             | Software - Infrastructure      | USA       |  1.77308e+12 | 25.9  |  240.33 |  -0.0059 | 17929705 |
+|  2 | GOOG     | Alphabet Inc.         | Communication Services | Internet Content & Information | USA       |  1.23434e+12 | 18.68 |   95.44 |  -0.0084 | 20145081 |
+|  3 | GOOGL    | Alphabet Inc.         | Communication Services | Internet Content & Information | USA       |  1.21605e+12 | 19.51 |   95.19 |  -0.009  | 20047848 |
+|  4 | AMZN     | Amazon.com, Inc.      | Consumer Cyclical      | Internet Retail                | USA       |  9.3352e+11  | 85.26 |   92.42 |  -0.0163 | 65245723 |
+...continued...
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/stocks-screener.md:247]: Please remove this continued statements and either do `...` or put continued at the bottom. It messes up the table.

File: website/content/sdk/guides/intros/stocks/stocks-screener.md [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+| 498 | LUMN     | Lumen Technologies, Inc. | Communication Services | Telecom Services              | USA       |     5.61e+09 |  2.74 |    5.47 |  -0.0091 | 31885469 |
+| 499 | PENN     | PENN Entertainment, Inc. | Consumer Cyclical      | Resorts & Casinos             | USA       |     5.35e+09 | 24.72 |   34.07 |   0.0119 |  1689121 |
+| 500 | NWL      | Newell Brands Inc.       | Consumer Defensive     | Household & Personal Products | USA       |     5.27e+09 |  9.6  |   12.87 |  -0.0191 |  5725896 |
+| 501 | VNO      | Vornado Realty Trust     | Real Estate            | REIT - Office                 | USA       |     4.86e+09 | 50.72 |   25.31 |   0.0218 |  2560781 |
+| 502 | PVH      | PVH Corp.                | Consumer Cyclical      | Apparel Manufacturing         | USA       |     4.21e+09 |  4.96 |   64.88 |   0.0399 |  1100532 |
+
+With this preset, the five different data types can be quickly assembled as one dataframe. Occasionally, an error may pop up when the screener is run; but, it should remedy itself if the command is repeated.
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/stocks-screener.md:254]: Please remove all info further from here. We don't have to inform the user that it is possible you receive an error. Next to that the sample code is too large and you already explain much of this above. We do not have to include this. The output also doesn't look as nice as the tables.

**Review by JerBouma — CHANGES_REQUESTED** [review]
Looking good, made some suggestions.

File: website/content/sdk/guides/intros/stocks/stocks-screener.md [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+|  4 | AMZN     |   92.42 |  -0.0163 | 65245723 |  9.3352e+11  | 85.26 |     55.08 |  3.28 |  1.86 |  6.85 | 15.91 |  N/A    |        0.549 |       0.8239 |        0.676  |        0.26   | 0.281           |
+
+Combined, the five DataFrames provide an outline of who they are, which segment of they market they belong to, and how they are currently trading.
+
+### Unconventional Applications
+
+One way to use the screener is to feed it the least amount of variables possible. In this next example, the preset file contains only one filter; requesting data for all constituents of the S&P 500 Index.
+
+```console
+# Author of preset: Danglewood
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/stocks-screener.md:217]: Please change this to OpenBB.

File: website/content/sdk/guides/intros/stocks/stocks-screener.md (hunk 3 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/stocks-screener.md:158]: Good catch.

File: website/content/sdk/guides/intros/stocks/stocks-screener.md (hunk 2 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/stocks-screener.md:94]: Removed the continued as the index numbers should provide some context.

File: website/content/sdk/guides/intros/stocks/stocks-screener.md (hunk 1 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/stocks-screener.md:57]: Done. ZIP file updated to reflect this as well.

File: website/content/sdk/guides/intros/stocks/stocks-screener.md (hunk 4 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/stocks-screener.md:247]: Done

File: website/content/sdk/guides/intros/stocks/stocks-screener.md (hunk 5 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/stocks-screener.md:254]: Done. Cut it out and cut straight to the chase with the sample script.

File: website/content/sdk/guides/intros/stocks/stocks-screener.md (hunk 6 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/stocks-screener.md:217]: Done and updated in the attached ZIP file

**Review by JerBouma — APPROVED** [review]

**Pull request merged**